test/unit/create-issue-test.js

```diff
@@ -53,10 +56,15 @@ test('create issue request fails', t => {
     api,
     debug: () => {},
     owner: 'owner',
-    repo: 'repo',
     branch: 'branch',
+    repo: null,
     sha: 'sha',
-    labels: 'labels',
+    config: {
+      repository: {
+        name: 'repo-name'
+      },
```

Comment:
see my comment above, this should be `repository: 'repo-name'`

Reply:
ohh yea, that was fixed before but looks like my rebase overrode it 😠  Fixing right now!